### PR TITLE
Remove comment about upcoming alerting for singlestat and table panels

### PR DESCRIPTION
**Pull request by theneva**
This might help with the general tone of voice in https://github.com/grafana/grafana/issues/6983

### CODE REVIEW

**Comment by stefanotorresi** [comment]
May I suggest rewording to:
```
Currently, only the *Graph* panel supports alert rules; support for other types of panels like *Singlestat* and *Table* may come in the future.
```
Also linking the open issue may be a good idea, but I'm new here so I don't know if it's common to have GH references in the docs.

**Comment by torkelo** [comment]
The current plan is to make alert rules it's own standalone thing (so not created as part of any panel or dashboard)

**Review by torkelo — APPROVED** [review]

**Comment by Zimza** [comment]
> The current plan is to make alert rules it's own standalone thing (so not created as part of any panel or dashboard)

Great news !
